data/systemd/anaconda-pre.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+# This service is to be run before anaconda starts and log data before anaconda changes them
+Description=pre-anaconda logging service
```

Comment:
Better description would be nice. More what is this service doing. This is nice to have.